### PR TITLE
Update the README

### PR DESCRIPTION
**Pull request by oddg**

### CODE REVIEW

**Comment by oddg** [comment]
@rogertsui0923 Ready for review

File: README.md [hunk 1]
```diff
@@ -1,113 +1,28 @@
...
- - vehicleRoutes: The optimized schedule
-
-Examples
---------
-Example 1:
+Remember to require it and set your token before using it
 
 ```ruby
 require 'routific'
+routific = Routific.set_token(--API_KEY--)
```
Comment (rogertsui0923) [inline README.md]: `set_token` doesn't return a Ruby instance; should it?

Comment (rogertsui0923) [reply README.md]: Currently we need to do:
```ruby
require 'routific'
Routific.set_token(--API_KEY--)
routific = Routific.new
...
```

Comment (oddg) [reply README.md]: You are right

**Comment by oddg** [comment]
@rogertsui0923 Just fixed the example.

**Pull request merged**